Scheduler/Pages/EditRecordPage.xaml

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Scheduler.Pages.EditRecordPage"
+             xmlns:statusConverter="clr-namespace:Scheduler.Converter;assembly=Scheduler">
```

Comment:
pay attention, you added namespace Converter and named it 'statusConverter' - that's can be confused for other converters on this page in future